includes/api/class-edd-api-v2.php

```diff
@@ -389,13 +389,17 @@ public function get_recent_sales() {
 				$user_info    = $payment->user_info;
 
 				$sales['sales'][ $i ]['ID']             = $payment->number;
+				$sales['sales'][ $i ]['mode']           = $payment->mode;
+				$sales['sales'][ $i ]['status']         = $payment->status_nicename;
```

Comment:
I think this would be better as `$payment->status`

Reply:
@ashleyfae, I went back-and-forth on which to do here. If someone was using the REST API with another integration just to quickly generate a report for stakeholders, not sure if "publish" or "edd_subscription" would mean as much as "Complete" or "Renewal". 

But, on the other side, most people who are interacting with the endpoint are probably developers/analysts who could filter/modify the status, if needed.

Yeah, probably better to just go with "status" then.

Reply:
Changed.

Reply:
Thanks for the change!

`status_nicename` is really the "display" value, whereas "status" is the internal one. I think the internal one is better for the API and on the other side people you can convert it to your desired display.